AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -102,6 +102,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 mylifebox.com##.welcome-page__cookie-notif
+globalhungerindex.org###ckpolicy
```

Comment:
This site has low visitors, but I guess `###ckpolicy` can be generic in the future.